src/assets/Main.php

```diff
@@ -27,6 +27,7 @@ class Main extends \luya\web\Asset
      */
     public $js = [
         'dist/js/main.js',
+        'https://cdn.bootcss.com/echarts/3.8.5/echarts-en.min.js',        
```

Comment:
We don't use CDN resources in the admin because we don't want to expose all the LUYA admin URLs to the respective CDN. Just integrate the JS via libs composer/bower (https://libraries.io/bower/echarts).